arroyo-worker/src/engine.rs

```diff
@@ -1138,6 +1138,20 @@ impl Engine {
                                     None
                                 }
                             }
+                            Some(ControlResp::Error {message, details}) => {
+                                info!("Handling worker error");
```

Comment:
this log line isn't that useful as is -- I'd either remove it, or have it log the details if they're not otherwise being logged

Reply:
Removed it